cmake/Helper.cmake

```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 set(WASMEDGE_INTERPROCEDURAL_OPTIMIZATION OFF)
-if(CMAKE_BUILD_TYPE STREQUAL Release OR CMAKE_BUILD_TYPE STREQUAL RelWithDebInfo)
+if((CMAKE_BUILD_TYPE STREQUAL Release OR CMAKE_BUILD_TYPE STREQUAL RelWithDebInfo) AND NOT WASMEDGE_FORCE_DISABLE_LTO)
```

Comment:
This is not correct. If the WASMEDGE_FORCE_DISABLE_LTO is `ON`, the `CMAKE_GENERATOR` will be ignored either.

Reply:
Fixed.